console-core-models.ts

```
@@ -1,6 +1,7 @@
-import { K8sModel as K8sKind } from '@console/dynamic-plugin-sdk/src/api/common-types';
+/* eslint-disable no-barrel-files/no-barrel-files */
```

ESLint

[error] Definition for rule 'no-barrel-files/no-barrel-files' was not found.

(no-barrel-files/no-barrel-files)